recipes/python-ghostscript/windll_path.patch

```diff
@@ -0,0 +1,11 @@
+--- python-ghostscript-v0.7.orig/ghostscript/_gsprint.py	2023-12-20 13:18:30.453497088 -0600
++++ python-ghostscript-v0.7/ghostscript/_gsprint.py	2023-12-20 13:45:31.513348016 -0600
+@@ -487,6 +487,8 @@
+     from distutils.version import LooseVersion
+     import os
+ 
++    return os.environ.get("GS_DLL", os.environ["CONDA_PREFIX"] + "\\library\\bin\\gsdll64.dll")
```

Comment:
You may need to pin ghostscript to ensure a new version, with a different DLL version number in it, won't break the package, right? I'm not an expert on Windows to know how that name can change.

Reply:
Thanks for the review!  On my windows test system, there is no versioning with the DLL that I can find.
![Screenshot from 2023-12-21 08-00-26](https://github.com/conda-forge/staged-recipes/assets/210858/8f10a8a0-d91e-4dd3-bbeb-b18655989988)